pylot/planning/rrt_star/rrt_star.py

```diff
@@ -0,0 +1,143 @@
+"""
+Author: Edward Fang
+Email: edward.fang@berkeley.edu
+This code is adapted from: https://github.com/dixantmittal/fast-rrt-star
+"""
+import networkx as nx
+from pylot.planning.rrt_star.utils import *
+
+
+def apply_rrt_star(state_space, starting_state, target_space, obstacle_map,
```

Comment:
I'm going to let it slip for now, but we should refactor this function in the future. It is 110 lines long :).

Reply:
Thanks :stuck_out_tongue: